shayne-shannon/scripts/article.js

```diff
@@ -0,0 +1,62 @@
+'use strict';
+
+var articles = [];
+
+function Article (rawDataObj) {
+  // DONE: Use the JS object passed in to complete this constructor function:
+  // Save ALL the properties of `rawDataObj` into `this`
+  this.title = rawDataObj.title,
+  this.category = rawDataObj.category,
+  this.author = rawDataObj.author,
+  this.authorUrl = rawDataObj.authorUrl,
+  this.publishedOn = rawDataObj.publishedOn,
+  this.body = rawDataObj.body
+}
+
+Article.prototype.toHtml = function() {
+  var $newArticle = $('article.template').clone();
+  $newArticle.removeClass('template').show();
```

Comment:
You do not need to have `.show()` here since the only reason the new article would have been hidden is due to the class of `template`.